src/main/resources/application.yaml

```diff
@@ -66,7 +66,7 @@ spring:
     profiles: local
 
 openshift:
-    cluster: qa
+    cluster: utv
```

Comment:
Should this be QA or utv? I am really not sure. These days i tend to test more in UTV really.

Reply:
The UTV that sort of is a production environment for us?

Reply:
We can discuss this in a metting later. Personally I would love to do more development related tasks in the qa cluster. Sometimes the ocp version differ so it is not that easy.